shared/logging_module.rb

```diff
@@ -46,6 +46,9 @@ def logger
             thread_id = " #{Thread.current[:thread_id]}" unless Thread.current[:thread_id].nil?
           end
 
+          if msg.class == Array
```

Comment:
Types 😍

Reply:
In my email this comment looks way better:

<img width="221" alt="screen shot 2018-03-07 at 11 58 38 am" src="https://user-images.githubusercontent.com/1304821/37115050-e9e49ffa-21fe-11e8-98a1-31a05f4e19ea.png">